src/main/webapp/script.js

```diff
@@ -90,53 +115,61 @@ async function displayOverallResults(url) {
  * @param {Array<sentimentBucket>} sentimentBucketList
  */
 function displaySentimentBucketChart(sentimentBucketList) {
-  const CommentSentimentTable = new google.visualization.DataTable();
-  CommentSentimentTable.addColumn('string', 'Sentiment Range');
-  CommentSentimentTable.addColumn('number', 'Comment Count');
-  CommentSentimentTable.addColumn(
-      {'type': 'string', 'role': 'tooltip', 'p': {'html': true}});
-
-  for (i = 0; i < sentimentBucketList.length; i++) {
-    currentSentimentBucket = sentimentBucketList[i];
-    rangeAsString = convertRangeToString(
-        currentSentimentBucket.intervalRange);
-    highestMagnitudeComments = currentSentimentBucket.topNComments;
-
-    CommentSentimentTable.addRow([rangeAsString,
-      currentSentimentBucket.frequency,
-      toTooltipString(highestMagnitudeComments)]);
+  try {
+    const CommentSentimentTable = new google.visualization.DataTable();
+    CommentSentimentTable.addColumn('string', 'Sentiment Range');
+    CommentSentimentTable.addColumn('number', 'Comment Count');
+    CommentSentimentTable.addColumn(
+        {'type': 'string', 'role': 'tooltip', 'p': {'html': true}});
+
+    for (i = 0; i < sentimentBucketList.length; i++) {
+      currentSentimentBucket = sentimentBucketList[i];
+      rangeAsString = convertRangeToString(
+          currentSentimentBucket.intervalRange);
+      highestMagnitudeComments = currentSentimentBucket.topNComments;
+
+      CommentSentimentTable.addRow([rangeAsString,
+        currentSentimentBucket.frequency,
+        toTooltipString(highestMagnitudeComments)]);
+    }
+
+    const options = {
+      'title': 'Comment Sentiment Range',
+      'width': CHART_WIDTH,
+      'height': CHART_HEIGHT,
+      'bar': {groupWidth: '100'},
+      'tooltip': {isHtml: true},
+    };
+
+    const view = new google.visualization.DataView(CommentSentimentTable);
+    const chart = new google.visualization.ColumnChart(
+        document.getElementById('chart-container'));
+    chart.draw(view, options);
+  } catch (err) {
+    throw new Error('Error in Displaying Chart');
```

Comment:
why are we doing throwing error instead of showing that "something goes wrong" on web page?

Reply:
It does show the "something goes wrong" message. The reason it throws this error is because the function that calls it will catch it and display the custom message.